lib/cocoapods/generator/app_target_helper.rb

```diff
@@ -99,8 +99,9 @@ def self.add_xctest_search_paths(target)
       # @return [void]
       #
       def self.add_swift_version(target, swift_version)
+        version_to_set = !swift_version.nil? ? swift_version : Validator::DEFAULT_SWIFT_VERSION
```

Comment:
not sure this belongs here, and if it does we need to update the documentation.

I still need to clearly understand a case in which in `validator.rb`

```ruby
    # @return [String] the SWIFT_VERSION to use for validation.
    #
    def swift_version
      return @swift_version if defined?(@swift_version)
      if version = dot_swift_version
        @swift_version = version
      else
        DEFAULT_SWIFT_VERSION
      end
    end
```

Can return `nil`? Is it if the contents of the `.swift-version` file are empty?

Reply:
if `.swift-version` is empty, it returns nil, and goes to DEFAULT_SWIFT_VERSION

Reply:
So yes I don't see the reason why we are doing a `nil?` check here and in the other place, @heyzooi